app/components/certificate/certificate-available.tsx

```diff
@@ -4,8 +4,8 @@ import Description from './description';
 import CertificateDisplay from './certificate-display';
 
 interface CertificateAvailableProps {
-  validFrom: Date;
-  validTo: Date;
+  validFrom: string;
```

Comment:
Since in other places we use dates as dates why you decided to change them to strings here?

Reply:
@Ririio If you format this in the parent component,  you sould probably use a separate name, it did send me for a spin too at the first time
Suggest validFromFormatted, validToFormatted

Reply:
You're right... It can cause problems if I use the same name format for two completely different data type. I'll be more careful for next time